src/styles.js

```diff
@@ -59,4 +61,21 @@ export const CropArea = styled('div')({
     borderLeft: 0,
     borderRight: 0,
   },
-})
+}
+const roundShape = {
+  borderRadius: '50%',
+}
+
+export const createCropArea = ({ shape, showGrid }) =>
```

Comment:
You can actually replace this function by a styled component using emotion props api: https://emotion.sh/docs/styled#object-styles

Like this, you would then only have to pass the props to `CropArea`:
```jsx
<CropArea
  cropShape={cropShape}
  showGrid={showGrid}
  style={{
    width: this.state.cropSize.width,
    height: this.state.cropSize.height,
  }}
  data-testid="cropper"
  />
```

and it would avoid recreating the whole emotion component at each render.